cmd/di.go

```diff
@@ -113,6 +118,10 @@ type Dependencies struct {
 	ServicesManager       *service.Manager
 	ServiceRegistry       *service.Registry
 	ServiceSessionStorage *session.StorageMemory
+
+	NATPinger           *traversal.Pinger
+	NATTracker          *traversal.EventsTracker
+	LastSessionShutdown chan bool
```

Comment:
bool channel indicates that there's 2 different possibilities here, when in reality there's only 1. Use struct{} instead.

Reply:
Does this need to be stored on the dependency struct?

Reply:
Also - what's the desired behaviour here in a multiple service case. Should it care which service is providing the session? If the last session turns out to be wireguard should we shut down openvpn or leave it hanging.